src/ptf/testutils.py

```diff
@@ -1858,6 +1858,35 @@ def verify_packets_any(test, pkt, ports=[], device_number=0):
 
     test.assertTrue(received == True, "Did not receive expected pkt on any of ports %r for device %d" % (ports, device_number))
 
+def verify_packet_any_port(test, pkt, ports=[], device_number=0):
```

Comment:
No it won't be the same. verify_any_packet_any_port has an issue, it will return as soon as a packet is received even the packet is not in the list of exp_packets, since the packet is not one of expected packet, an error will be throw out. Any background traffic noise could cause the test fail. 

the new verify_packet_any_port, will return only when an expected packet is received or timeout. background traffic noise won't cause the test to fail.

We could fix verify_any_packet_any_port to ignore background traffic in the future. I thought it's nice to have a function to one packet on any port verification, as it will be used extensively for ecmp/lag validation.